package.json

```
@@ -122,6 +123,7 @@
     "@types/react-native-i18n": "^2.0.0",
     "@types/react-native-loading-spinner-overlay": "^0.5.1",
     "@types/react-native-push-notification": "^3.0.5",
+    "@types/react-native-share": "^3.1.0",
```

```suggestion
    "@types/react-native-share": "^3.3.2",
```
why not the last version? https://www.npmjs.com/package/react-native-share

---

maybe just an error in C&p the installation command online, I'll update it in next commit

---

@Undermaken just a moment maybe you confused the `react-native-share` package with his related `@types` the package is already at version 3.3.2

---

@CrisTofani 
Yes! Sorry, my bad!